idemp-rest-test/testing/src/utils/Utils.ts

```diff
@@ -50,13 +56,44 @@ export async function createPatient(valueObject: ValueObject) {
 	}
 }
 
+/**
+ * Create a supplier. If a business partner already exists on the value object, this won't do anything.
+ * @param valueObject The value object containing information to create the entity
+ * @returns Nothing
+ */
+export async function createVendor(valueObject: ValueObject){
+	valueObject.validate();
+
+	if (!valueObject.businessPartner) {
+		const vendor: Partial<Vendor> = {
+			name: valueObject.getDynamicStepMessage(),
+			description: valueObject.getStepMessageLong(),
+		};
+		valueObject.businessPartner = await vendorApi.save(valueObject, vendor as Vendor);
+		if (!valueObject.businessPartner) {
+			throw new Error('Vendor not created');
+		}
+		delete (valueObject.businessPartner as Partial<Patient>).approximateDateOfBirth;
+	}
+}
+
 /**
  * Create a business partner (don't really have an ideal method for this at the moment - have to go through patients).
  * If a business partner already exists on the value object, this won't do anything.
  * @param valueObject The value object containing information to create the entity
  * @returns Nothing
  */
-export async function createBusinessPartner(valueObject: ValueObject) {
+export async function createBusinessPartnerAsVendor(valueObject: ValueObject) {
```

Comment:
Can you update the method documentation since this method isn't going through patients?

Reply:
Updated